qcengine/programs/util/pdict.py

```diff
@@ -0,0 +1,112 @@
+from decimal import Decimal, ROUND_FLOOR, ROUND_CEILING
+
+import numpy as np
+
+# This file has considerable history in the quantum chemistry package Psi4
```

Comment:
Having a util folder and a `util.py` might be getting out of hand.

Reply:
I agree. But these are internal (non exported) utils. What about renaming the outer util “executing” in keeping with recent trends? (Would need a qcel bump)